Firebase/Auth/Source/FIRAuthWebView.m

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2017 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRAuthWebView.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation FIRAuthWebView
+
+- (instancetype)initWithFrame:(CGRect)frame {
+  self = [super initWithFrame:frame];
+  if (self) {
+    self.backgroundColor = [UIColor whiteColor];
+    [self initializeSubviews];
+  }
+  return self;
+}
+
+- (void)initializeSubviews {
+  UIWebView *webView = [self createWebView];
+  UIActivityIndicatorView *spinner = [self createSpinner];
+
+  // The order of the following controls z-order.
```

Comment:
Not a huge deal but we deviate from the style of comments we have been using to document methods in this file, we can simple use these messages as the brief of the style we usually use.

This may be put of scope for this pull request.

Reply:
Done.